pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Just as your schemas evolve over time, the JSON Schema specification does as well. This guide provides useful information to make upgrading your schemas across versions of the specification a breeze.
+
+Learn how to use our migration guides and tooling ([AlterSchema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you started.
+
+| All Keywords                | Specification | Draft introduction | Removed | Changed                                                                                      |
+| --------------------------- | ------------- | ------------------ | ------- | -------------------------------------------------------------------------------------------- |
+| `$anchor`                   | Core          | 2019-09            | No      | Partially replaced `$id`                                                                     |
+| `$comment`                  | Core          | 07                 | No      |                                                                                              |
+| `$id`                       | Core          | 06                 | No      | Replaced `id`                                                                                |
+| `$defs`                     | Core          | 2019-09            | No      | Replaced `definitions`                                                                       |
+| `$dynamicAnchor`            | Core          | 2020-12            | No      | Replaced `$recursiveAnchor`                                                                  |
+| `$dynamicRef`               | Core          | 2020-12            | No      | Replaced `$recursiveRef`                                                                     |
```

Comment:
Hi, I think because `$dynamicRef`, `$dynamicAnchor` added more flexibility / usage over $`recursiveAnchor`.
Maybe we can say it as replacement or enhancement ? OR something like that